.github/workflows/update-image-digest.yml

```diff
@@ -109,3 +109,15 @@ jobs:
           labels: automated pr, kind/cleanup, release-note-none
           branch: update-digests
           delete-branch: true
+
+      - name: Get previous job's status
+        id: lastrun
+        uses: filiptronicek/get-last-job-status@main
+      - name: Slack Notification
+        if: ${{ (success() && steps.lastrun.outputs.status == 'failed') || failure() }}
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.WORKSPACE_SLACK_WEBHOOK }}
```

Comment:
@iQQBot can we send to #substrate-release-alerts, instead? I ask because this impacts release assets from a security standpoint for Enteprise and PAYG.

Adding approval to unblock, this is certainly better than nothing, it's just that #enterprise-jobs may not receive 👀 as quickly.